pkgs/build-support/setup-hooks/make-symlinks-relative.sh

```diff
@@ -0,0 +1,28 @@
+fixupOutputHooks+=(_makeSymlinksRelative)
```

Comment:
wlroots has 
```
wlroots> +++ find /nix/store/3a0xwszw8n5dzzhsgfnilvsqi4hk565s-wlroots-0.15.1-examples -type l -print0
wlroots> find: '/nix/store/3a0xwszw8n5dzzhsgfnilvsqi4hk565s-wlroots-0.15.1-examples': No such file or directory
```

because the examples output is created in postFixup while this hook runs in fixupPhase

if this is changed to `postFixupHooks` then we shouldn't get a error like that in many builds

Reply:
#204692